clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -254,14 +254,14 @@ public NetworkClientDelegate.UnsentRequest toUnsentRequest() {
                     new ArrayList<>(this.requestedPartitions),
                     throwOnFetchStableOffsetUnsupported);
             return new NetworkClientDelegate.UnsentRequest(
-                    builder,
-                    coordinatorRequestManager.coordinator(),
-                    (r, t) -> onResponse(r.receivedTimeMs(), (OffsetFetchResponse) r.responseBody()));
+                builder,
+                coordinatorRequestManager.coordinator(),
+                (r, t) -> onResponse(r.receivedTimeMs(), (OffsetFetchResponse) r.responseBody()));
```

Comment:
Out of curiosity, can the response never be `null` here as in `HeartbeatRequestManager`? Can timeouts not happen here?

Reply:
In fact, this is a bug and I'm planning to address this in [KAFKA-15562](https://issues.apache.org/jira/browse/KAFKA-15562).  Can we address the gap all in a follow up PR?

Reply:
Of course, mine was a question for my understanding.